package.json

```
@@ -18,13 +18,12 @@
     "tsc-installer": "tsc -p ./src/install/tsconfig.json",
     "doc": "node utils/doclint/cli.js",
     "test-infra": "node utils/doclint/check_public_api/test/test.js && node utils/doclint/preprocessor/test.js && node utils/testrunner/test/test.js",
-    "lint": "npm run eslint && npm run tsc && npm run doc && npm run check-deps && npm run test-types && npm run test-infra",
-    "debug-test": "node --inspect-brk test/test.js",
-    "clean": "rimraf lib && rimraf types",
+    "lint": "npm run eslint && npm run tsc && npm run doc && npm run check-deps && npm run generate-types && npm run test-types && npm run test-infra",
+    "clean": "rimraf lib",
     "prepare": "node install-from-github.js",
-    "build": "node utils/runWebpack.js --mode='development' && tsc -p . && npm run generate-types",
+    "build": "node utils/runWebpack.js --mode='development' && tsc -p .",
     "watch": "node utils/runWebpack.js --mode='development' --watch --silent | tsc -w -p .",
-    "test-types": "npm run generate-types && npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
+    "test-types": "npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
```

```suggestion
    "test-types": "npm run generate-types && npx -p typescript@3.7.5 tsc -p utils/generate_types/test/tsconfig.json && npm run typecheck-tests",
```

Can we put generate-types back here? Not having it breaks my muscle memory for working with the types.

---

Oh I see, it exits 1 if generating it fails.. That's unfortunate.